js/layout-javascript/small-card-code.js

```diff
@@ -383,17 +391,13 @@ DynamicList.prototype.attachObservers = function() {
       _this.$container.find('.clear-filters').removeClass('hidden');
     })
     .on('click', '.list-search-cancel', function() {
-      var $elementClicked = $(this);
-      var $parentElement = $elementClicked.parents('.new-small-card-list-container');
-
-      _this.Utils.Page.updateFilterControlsContext();
-
-      if ($parentElement.find('.hidden-filter-controls').hasClass('active')) {
-        $parentElement.find('.hidden-filter-controls').removeClass('active');
-        $elementClicked.removeClass('active');
-        $parentElement.find('.list-search-icon .fa-sliders').removeClass('active');
-        $parentElement.find('.hidden-filter-controls').animate({ height: 0 }, 200);
-      }
+      // Hide filters
+      $(this).removeClass('active');
+      _this.$container.find('.hidden-filter-controls').removeClass('active');
+      _this.$container.find('.list-search-icon .fa-sliders').removeClass('active');
+      _this.$container.find('.hidden-filter-controls').animate({ height: 0 }, 200);
+      // Clear filters
```

Comment:
empty line before comments

Reply:
Thanks. There's another PR coming soon so unique identifiers are customisable. I'll add the empty lines then.

Reply:
The same for other files